polkadot-parachains/src/chain_spec.rs

```diff
@@ -141,6 +141,7 @@ fn chain_spec<F: Fn() -> GenesisConfig + 'static + Send + Sync, GenesisConfig>(
 		// protocol id
 		Some(&format!("nctr-{}", relay_chain.to_string().chars().nth(0).unwrap())),
 		// properties
+		None,
```

Comment:
Added `fork_id = None`, don't know exactly for what it is, but None seems reasonable given the name.

Reply:
Edit, this is the doc; None is fine.

```
	/// Arbitrary string. Nodes will only synchronize with other nodes that have the same value
	/// in their `fork_id`. This can be used in order to segregate nodes in cases when multiple
	/// chains have the same genesis hash.
```